examples/fluids/src/misc.c

```diff
@@ -394,3 +394,15 @@ PetscErrorCode PHASTADatFileReadToArrayReal(MPI_Comm comm, const char path[PETSC
 
   PetscFunctionReturn(0);
 }
+
+PetscLogEvent       FLUIDS_CeedOperatorApply;
+static PetscClassId libCEED_classid;
+
+PetscErrorCode RegisterLogEvents() {
+  PetscFunctionBeginUser;
+
+  PetscCall(PetscClassIdRegister("libCEED", &libCEED_classid));
+  PetscCall(PetscLogEventRegister("CeedOperatorApp", libCEED_classid, &FLUIDS_CeedOperatorApply));
```

Comment:
Let's get the others in this PR while we're at it.
```suggestion
  PetscCall(PetscLogEventRegister("CeedOpApply", libCEED_classid, &FLUIDS_CeedOperatorApply));
    PetscCall(PetscLogEventRegister("CeedOpAsm", libCEED_classid, &FLUIDS_CeedOperatorAssemble));
    PetscCall(PetscLogEventRegister("CeedOpAsmD", libCEED_classid, &FLUIDS_CeedOperatorAssembleDiagonal));
    PetscCall(PetscLogEventRegister("CeedOpAsmPBD", libCEED_classid, &FLUIDS_CeedOperatorAssemblePointBlockDiagonal));
```